CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * Mixed property can now be indexed ([#4342](https://github.com/realm/realm-core/issues/4342))
- 
+* Dictionary of Object need to be nullable, but that would lead to inconsistency in schema and cause an exception ([#4344](https://github.com/realm/realm-core/issues/4344))
```

Comment:
Although we know what you mean, I'd prefer if we more explicitly state that it's the value part of a key-value pair which is nullable, and not a complete "dictionary of objects".

Reply:
@finnschiermer better now?